FinanceSchedulerDemo/FinanceSchedulerDemo/ClientApp/src/app/nav-menu/nav-menu.component.html

```diff
@@ -25,6 +25,7 @@
         class="navbar-collapse collapse d-sm-inline-flex justify-content-end"
         [ngClass]="{ show: isExpanded }"
       >
+        <div [ngClass]="{visible: (currentEnvironment == 'Development'), hidden: (currentEnvironment == 'Staging')}"> Development mode ON </div>
```

Comment:
Can you use bootstrap classes?
And it seems that it's possible to use only one condition